### ISSUE TITLE
My media player is not detected

### ISSUE THREAD

**Issue by ricoshett** [ungive/discord-music-presence#58]
Players I am using, that aren't supported yet:
```
version: 2.2.4
win_winrt: ru.yandex.desktop.music - ru.yandex.desktop.music (disabled)
signature: 1d136aa9f70e06c5
```

---

**Comment by ungive** [comment]
Added support for Yandex.Music, thanks for reporting!

**Comment by ungive** [comment]
could you share your logs? they're located at `%APPDATA%\Music Presence\presence.log`

**Comment by ungive** [comment]
thanks, which song is not detected and at what time? note that the timestamps in the logs are in UTC. speaking by the part of the logs you sent it looks like the song is displayed in your status

**Comment by ungive** [comment]
I'm not really sure why one would be showing and not the other, especially if the logs say that the Discord status has been set. Could you share more of your logs where you switch between Soundcloud and Yandex.Music? What setting do you have for "Show the name of the player instead of Music" in the Appearance settings? Can you try unchecking it (if it isn't already) and see if switching between the two works then?

**Comment by ungive** [comment]
can you try playing the exact same song? e.g. "Eminem - Superman" with Yandex.Music. it might be that the cyrilic/unicode letters cause issues

**Comment by ungive** [comment]
I'll try it myself once I get around to it!

**Comment by ungive** [comment]
Yeah that can definitely be! Whenever a different name is shown ("Listening to Yandex.Music", "Listening to Soundcloud", etc.) it has to reconnect to Discord under the hood and that can take some time. Usually just a few seconds though.
